scripts/e2e/cmd/runner/testdata/one-namespace-one-ingress/health-probe-configurations/app.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-deployment
+spec:
+  selector:
+    matchLabels:
+      app: test-app
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: test-app
+    spec:
+      containers:
+        - name: test-app
+          imagePullPolicy: Always
+          image: docker.io/kennethreitz/httpbin
+          ports:
+            - containerPort: 80
+          livenessProbe:
+            httpGet:
+              path: /statusx/200
+              port: 80
+              host: localhostx
```

Comment:
did you add "x" so that the readiness probe always fail ?

Reply:
correct